x/cellarfees/types/params.go

```diff
@@ -14,16 +25,109 @@ func ParamKeyTable() paramtypes.KeyTable {
 
 // DefaultParams returns default cellarfees parameters
 func DefaultParams() Params {
-	return Params{}
+
+	return Params{
+		// Rough number of blocks in 2 weeks, or ~2 fee accrual cycles for one cellar
+		AuctionBlockDelay: 201600,
+		// Rough number of blocks in 28 days, the time it takes to unbond
+		RewardEmissionPeriod: 403200,
+		// Initial rate at which an auction should decrease the price of the relevant coin from it's starting price
+		InitialPriceDecreaseRate: sdk.NewDec(347000000000000),
```

Comment:
This seems like a kind of bizarre way to set a percentage. Maybe it's just the way `Dec` works, but is there a way to make this clearer?

Reply:
Yeah, `Dec` is a big.Int that considers the first (little end) 18 digits to be decimal places. I think the param could be a float and we'd just convert it when we do the emission math?